cmd/version/main.go

```diff
@@ -3,22 +3,34 @@ package version
 import (
 	"fmt"
 
-	"github.com/datreeio/datree/bl"
+	"github.com/datreeio/datree/bl/messager"
 	"github.com/spf13/cobra"
 )
 
+type Messager interface {
+	LoadVersionMessages(messages chan *messager.VersionMessage, cliVersion string)
+}
+
+type Printer interface {
+	PrintMessage(messageText string, messageColor string)
+}
 type VersionCommandContext struct {
-	CliVersion           string
-	VersionMessageClient bl.VersionMessageClient
+	CliVersion string
+	Messager   Messager
+	Printer    Printer
 }
 
 func version(ctx *VersionCommandContext) {
-	messageChannel := bl.PopulateVersionMessageChan(ctx.VersionMessageClient, ctx.CliVersion)
+	messages := make(chan *messager.VersionMessage, 1)
```

Comment:
we can only have one message. I think this assumes that each version supports multiple messages which we don't at the moment.
Not a big issue, just pointing it out

Reply:
as much as I know this is the best way to handle 1 buffered channels (the last argument `1` indicates that this channel is allocated to only one member)